src/main/java/it/grid/storm/info/du/DiskUsageUtils.java

```diff
@@ -0,0 +1,71 @@
+package it.grid.storm.info.du;
+
+import static it.grid.storm.space.ExitCode.EMPTY_OUTPUT;
+import static it.grid.storm.space.ExitCode.FAILURE;
+import static it.grid.storm.space.ExitCode.IO_ERROR;
+import static it.grid.storm.space.ExitCode.SUCCESS;
+
+import java.util.Date;
+import java.util.List;
+
+import it.grid.storm.space.DUResult;
+import it.grid.storm.space.ExitCode;
+
+public class DiskUsageUtils {
+
+  private DiskUsageUtils() {}
+
+  private static DUResult getResult(ExitCode code, String absPath, long size, long start, long end) {
```

Comment:
It would be safer to use new classes from java.time to handle intervals here...

Reply:
done